src/BlazorWasmSdk/Tasks/GenerateBlazorWebAssemblyBootJson.cs

```diff
@@ -44,6 +44,8 @@ public class GenerateBlazorWebAssemblyBootJson : Task
 
         public string Jiterpreter { get; set; }
 
+        public string RuntimeOptions { get; set; }
```

Comment:
Hmm, this is coming from a property from what I can tell. If you have multiple values, shouldn't it be better if it comes from an ItemGroup?

I believe it still can be a property if you use `;` as the delimiter and just evaluate it in the itemgroup as Include="$(Prop) and MSBuild will do the parsing for you.

Reply:
@rainersigwald is there a way to go from Property to Item using an alternate character separator?

Reply:
The reason a choosed property is that it has a syntax of command line arguments. So the user types "-a -b -c". But I can change it to item group if that is a preferred way.

Reply:
@maraf It is a bit strange to use the command-line syntax for the symbol in MSBuild.

It means that if you ever change the CLI syntax, then you break anybody that had the option setup in the csproj file. Typically, the SDK defines well-known properties for the different options and those get translated later to specific CLI flags/arguments.

I am not asking that you change it, just that you think about it, if you want to be consistent with other areas of the product.

Reply:
I prefer opaque style.

I think this is very low level feature and that SDK should not create tight coupling to Mono VM features.
If the customer project already knows enough about Mono runtime options, they are already tightly coupled to it.

Also we are not trying to document the full list of options here. 
We are not making individual options public API here.

Reply:
@javiercn I see you point and I would go that way. I asked around whether it's worth to intruducing property per runtime option and the answer I got is that once jiterpreter is stabilized, we won't need these settings. The main reason we introduce this property now is to be able to granularily disable jiterpreter features in following previews.

Reply:
@pavelsavara For features that are used quite often, I prefer to have concrete properties, and once we will have these bits of SDK closer to runtime, it would be easy to do.

Reply:
> @rainersigwald is there a way to go from Property to Item using an alternate character separator? I know for displaying you can use `@(Item, ', ')` to define an alternative separator, but I don't know a way to do it the other way around.

Sorry, I was out of the office when this was fresh. There's no way to have the MSBuild engine do that for you, just "task takes `string` and splits on its own delimiter".